src/widgets/hero-course/ui/hero-course.module.scss

```diff
@@ -0,0 +1,40 @@
+.hero-course {
+  height: 55vh;
```

Comment:
Is this an intentional change?
![image](https://github.com/user-attachments/assets/42727ae5-ee7d-4859-ba7c-ac366ac92117)

Reply:
What is the screenshot on the left? Deploy looks more like the one on the right now. I didn't change the height, I just added `vh`

Reply:
> What is the screenshot on the left? Deploy looks more like the one on the right now. I didn't change the height, I just added `vh`

And that's the reason why it looks different on my monitor

Reply:
It doesn't look very good. Should I just remove the height in `vh` or is there a better solution?

Reply:
Yeah, let's just remove it

Reply:
Done